composer.json

```diff
@@ -26,7 +26,9 @@
     },
     "require": {
         "php":             ">=5.4.0",
-        "symfony/symfony": ">=3.0"
+        "symfony/framework-bundle": ">=3.0",
```

Comment:
Maybe using `^3.0||^4.0` would be better?

Reply:
You're right - changing